### PR TITLE
[Feat] Basic responsive Navbar

### PR DESCRIPTION
**Pull request by rushikesh3299**
Animation

### CODE REVIEW

**Review by JantuDeb — APPROVED** [review]
This is really good that you have used media query for responsiveness

File: Components/Nav/nav.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+            <a href="/Components/Image/image.html" class=" p20">Image</a>
+            <a href="/Components/Input/input.html" class=" p20">Input</a>
+            <a href="/Components/Rating/rating.html" class=" p20">Rating</a>
+            <a href="/Components/Toast/toast.html" class=" p20">Toast</a>
+            <a href="/Components/Typography/typography.html" class=" p20">Typography</a>
+        </div>
+        <div class="component-section">
+            <div class="component-title">Navigation Bar</div>
+            <p class="component-general">Navigation bar is used to navigate through the web apps or websites.</p>
+            <div class="component-subtitle">Example of Navbar</div>
```
Comment (ajit-sharmaGH) [inline Components/Nav/nav.html:46]: Good naming convention

**Review by ajit-sharmaGH — APPROVED** [review]
Looks good 
Clean code 
You can merge it

**Review by shuklahimanshu81 — APPROVED** [review]
Your naming conventions are impressive will try to implement them in upcoming parts of my project. And also the code structure is super clean. Good to merge.

**Pull request merged**